content/en/user-manual/graphics/lighting/clustered-lighting.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Clustered Lighting
+layout: usermanual-page.hbs
+position: 6
+---
+
+Lights are a great way to add realism to your application. However, real time lights can also create significant runtime performance cost, especially when you have large numbers of lights that cast shadows.
+
+Part of the solution to reduce performance costs may involve limiting the amount of lights that affect individual meshes. This is often implemented by finding and using lights that are nearby each object. However, there are multiple disadvantages to this strategy:
+- Since each object can use different sets of lights, a custom shader must be compiled to handle them.
+- Large objects need to be split into smaller objects in order for this strategy to be effective.
+- Large numbers of shadow casting lights can cause the shader to run out of the available texture slots used by shadow maps.
+
+To address these issues, PlayCanvas uses the **Clustered Lighting** solution to provide a performant implementation of Omni Lights and Spot Lights. It stores information about the lights in textures and allows the GPU to easily use only the lights that are nearby the shaded fragment. There are multiple advantages to Clustered Lighting:
+- Shaders do not need to be recompiled when lights are added or removed from the scene, since the shader can handle multiple lights.
+- Large numbers of lights (including Shadows and Cookies) can be used in the scene, since only the lights nearby each pixel are evaluated.
+
+Note that Directional Lights affect all objects so they do not use the Clustered Lighting solution.
+
+<br/>
+
+
+
```

Comment:
I'm not a fan of these blocks of whitespace + `br` tags. If you feel that the content spacing is bad, we should tweak CSS. This would ensure the _entire_ developer site is consistent in terms of paragraph spacing.